core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2622,6 +2622,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_NETWORK_NETTY_WRITER_FLUSH_TIMEOUT_MS =
+      new Builder(Name.USER_NETWORK_NETTY_WRITER_FLUSH_TIMEOUT_MS)
+          .setDefaultValue("12hour")
```

Comment:
Is this kind of long? Maybe 30 minutes? Also, you do not need to suffix with `MS`

Reply:
30 minutes may not be enough for a slow network. FLUSH time in worst case is nearly the same as the whole upload time for S3 Streaming upload. The whole upload time for a big file is reasonably to be hours. 12 hour may be too long, but 30 minutes may not be enough. May be 3 or 6 hours?

Reply:
My reasoning is that our previous close timeout is 30 minutes.

Reply:
If users want to upload a big file...we suggest them configure themselves. Will change to 30 minutes, thanks